docs/source/index.rst

```diff
@@ -31,6 +31,7 @@ Contents
    serialization
    searching
    customization
+   conventions
 
 .. include:: ../../README.rst
 
```

Comment:
Including the README into the docs is **not** best practice ! :)

There is a difference between a README and documentation !
Also with you current README you will include for example all the widgets ov travis and so oninto you docs, this is not really cool or a benefit.